.github/workflows/deb.yml

```diff
@@ -0,0 +1,79 @@
+name: deb
+on:
+  pull_request:
+    branches: [ master ]
+jobs:
+  release:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Generate variables
+      id: gen_vars
+      run: |
+        echo "::set-output name=short_sha::$(echo ${GITHUB_SHA} | cut -c1-8)"
+        echo "::set-output name=cur_datetime::$(date '+%Y-%m-%d %H:%M:%S')"
+    - name: Create a Release
+      uses: actions/create-release@v1
+      id: create_release
+      env:
+        GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
+      with:
+        tag_name: build-${{ github.sha }}
+        release_name: DRAKVUF ${{ steps.gen_vars.outputs.cur_datetime }} ${{ steps.gen_vars.outputs.short_sha }}
+        owner: tklengyel
+        repo: drakvuf-builds
+        draft: false
+        prerelease: false
+    - name: Store release URL as file
+      run: |
+        echo "${{ steps.create_release.outputs.upload_url }}" > release-url.txt
+    - name: Upload release URL artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: release-url
+        path: release-url.txt
+  deb:
+    needs: [release]
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+          - "debian:buster"
+          - "ubuntu:bionic"
+          - "ubuntu:focal"
+    steps:
+      - name: Fetch release URL
+        uses: actions/download-artifact@v1
+        with:
+          name: release-url
+      - name: Generate variables
+        id: gen_vars
+        run: |
+          SYSTEM_ID=$(echo ${{ matrix.container }} | cut -f1 '-d:')
+          DISTRIBUTION=$(echo ${{ matrix.container }} | cut -f2 '-d:')
+          ARTIFACT_PREFIX="${SYSTEM_ID}_${DISTRIBUTION}"
+          echo "Artifact prefix: $ARTIFACT_PREFIX"
+          RELEASE_URL=$(cat release-url/release-url.txt)
+          echo "Release URL: $RELEASE_URL"
+          echo "::set-output name=artifact_prefix::${ARTIFACT_PREFIX}"
+          echo "::set-output name=release_url::${RELEASE_URL}"
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Build package
+        id: build_drakvuf_deb
+        run: |
+          sh package/build.sh ${{ matrix.container }}
+          DRAKVUF_DEB_PATH=$(find package/out/*.deb | head -n1)
+          DRAKVUF_DEB_NAME=$(basename "$DRAKVUF_DEB_PATH")
+          echo "::set-output name=drakvuf_deb_path::${DRAKVUF_DEB_PATH}"
+          echo "::set-output name=drakvuf_deb_name::${{ steps.gen_vars.outputs.artifact_prefix }}_${DRAKVUF_DEB_NAME}"
+      - name: Upload release asset
+        uses: actions/upload-release-asset@v1.0.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
```

Comment:
You would need to create a secret that allows to access `tklengyel/drakvuf-builds` and then in GitHub repository settings for `tklengyel/drakvuf` add this secret as `RELEASE_TOKEN`, then restart the Workflow and it should work.

Creating secret for GH Actions:

https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token

> If you need a token that requires permissions that aren't available in the GITHUB_TOKEN, you can create a personal access token and set it as a secret in your repository:
> 
> 1. Use or create a token with the appropriate permissions for that repository. For more information, see "Creating a personal access token."
> 2. Add the token as a secret in your workflow's repository, and refer to it using the ${{ secrets.SECRET_NAME }} syntax. For more information, see "Creating and using encrypted secrets."

Reply:
done